public/app/features/alerting/unified/components/rules/ThresholdRules.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { LoadingPlaceholder, useStyles } from '@grafana/ui';
+import React, { FC, useMemo } from 'react';
+import { useUnifiedAlertingSelector } from '../../hooks/useUnifiedAlertingSelector';
+import { RulesGroup } from './RulesGroup';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
+import { RuleNamespace } from 'app/types/unified-alerting';
+import { initialAsyncRequestState } from '../../utils/redux';
+
+export const ThresholdRules: FC = () => {
+  const styles = useStyles(getStyles);
+  const { loading, result } = useUnifiedAlertingSelector(
+    (state) => state.rules[GRAFANA_RULES_SOURCE_NAME] || initialAsyncRequestState
+  );
+
+  const namespaces = useMemo(
+    (): RuleNamespace[] =>
+      (result || [])
+        // sort groups within namespace
+        .map((namespace) => ({
+          ...namespace,
+          groups: namespace.groups.slice().sort((a, b) => a.name.localeCompare(b.name)),
+        }))
+        // sort namespaces
+        .sort((a, b) => a.name.localeCompare(b.name)),
+    [result]
+  );
+
+  return (
+    <section className={styles.wrapper}>
+      <div className={styles.sectionHeader}>
+        <h5>Threshold</h5>
+        {loading ? <LoadingPlaceholder className={styles.loader} text="Loading..." /> : <div />}
+      </div>
+
+      {namespaces?.map((namespace) =>
+        namespace.groups.map((group) => (
```

Comment:
Asking more for my own knowledge here but what is the concept of a group and namespace for threshold alerts? I was only aware of folders for groups

Reply:
threshold rules have only folders, but API shape has to be the same for both kinds. So I believe in pracitce we'll map folders to namespaces and have dummy groups for grafana managed alerts